### ISSUE TITLE
resalloc: Unknown pool

### ISSUE THREAD

**Issue by xsuchy** [fedora-copr/copr#2983]
In the resalloc log I see:
```
ERROR 2023-11-03 15:53:13,939 PID:316334:139871599429440(Manager) Unknown pool aws_x86_64_powerful_normal_prod to cleanup
ERROR 2023-11-03 15:53:13,948 PID:316334:139871599429440(Manager) Unknown pool aws_aarch64_powerful_normal_prod to cleanup
```
Not sure if this is Copr's miscongiruation or resalloc issue.

**Comment by praiskup** [comment]
Seems like `aws_x86_64_powerful_spot_prod` vs. `aws_x86_64_powerful_normal_prod` problem.  The "normal" seems new in "powerful" category.  This is related to a hotfix for https://github.com/praiskup/resalloc/issues/128 - but it seems obvious we should have a better error message.

**Comment by praiskup** [comment]
Btw., `aws_x86_64_powerful_normal_prod` is not going to allocate resources (normally), since it has lower priority compared to `aws_x86_64_powerful_spot_prod`.  Even if EC2 refuses to reserve spot instances for us, resalloc will continue asking for them never falling back to the "normal" category.

Normal high-performance builders will only be allocated if all the spot builders are already requested.

**Comment by praiskup** [comment]
Soft-blocked on this: https://pagure.io/fedora-infrastructure/issue/11602 I'll continue tomorrow.

**Comment by praiskup** [comment]
New resalloc deployed to production.
